tests/Homely.AspNetCore.Mvc.Helpers.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:6417/",
+      "sslPort": 44360
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "Homely.AspNetCore.Mvc.Helpers.Tests": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    }
+  }
+}
```

Comment:
mistake? tests project shouldn't have launchSettings?

Reply:
Nope - it's a 'feature' of VS. the project is a `netcoreapp` so it includes that when doing stuff in VS. I did it once before and I had to remove it. So it accidentally popped back in. Maybe from a nuget update?

removed and also the `properties` folder removed from csproj.